server/odc-common/src/main/java/com/oceanbase/odc/common/util/SilentExecutor.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.common.util;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author yaobin
+ * @date 2024-03-25
+ * @since 4.2.4
+ */
+@Slf4j
+public class SilentExecutor {
```

Comment:
why should we need this?

Reply:
see above comment, to handle exception in for loop